lib/gollum-lib/macro/video.rb

```diff
@@ -0,0 +1,17 @@
+module Gollum
+  class Macro
+    class Video < Gollum::Macro
+      def render (fname)
+        begin 
```

Comment:
Thanks, I will plan to incorporate the changes. Moreover, after reading the html5 spec and also based on the information from moziila here https://developer.mozilla.org/en-US/docs/Learn/HTML/Multimedia_and_embedding/Video_and_audio_content, we don't really need to worry about mime types unless we want to implement some sort of optimization logic in our gollum macro as per the html spec (refer to the <source> element and its 'type' attribute). Here is the quote from the spec:  

"The type attribute of the source element allows the user agent to avoid downloading resources that use formats it cannot render."
> 

This idea elaborated further in the mozill'a documentation I linked to above. So, the usage of 'type/mime' in this context has only meaning when using multiple <source> elements giving the browser a chance to skip sources with mime types it doesn't support. According to the spec, if you don't specify a mime type for a source, the browser will need to download it first to determine the mime. Then skip to the next source, if needed to, and so forth.  So the 'type' attribute is just a hint to the browser and thus optional.  But for performance reasons, it should be used when linking to a video with multiple sources.  For wiki users it also would be simpler to just specify the file name without being bothered with mime types.